doc/index.rst

```diff
@@ -1,10 +1,44 @@
-The Plutus Platform
+Plutus Tools SDK User Guide
 ===============================
 
-The Plutus Platform is an application development platform for the Cardano blockchain.
-To read more about the Platform, see :ref:`what_is_the_plutus_platform`.
-To get started using the Platform see :ref:`plutus_getting_started`.
+Intended Audience
+-------------------------------
 
+External Developers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Isn't it redundant to have a separate section for "External developers" when this doc is specifically for external developers?

Reply:
Also, this is pretty weird to use in an users guide! From the users guide perspective "normal" developers are "external", we're weird for being "internal"!

Reply:
We could probably go without the "External Developers" heading. My goal is that if a reader who knew nothing about any of this came to this page for the first time, I want them to understand what this documentation is, who it's for, why they may or may not care about it. Easy to understand and clear.